docs/tools/index.md

```diff
@@ -47,7 +47,7 @@ Use Helm to:
 
 ##### Kompose 
 
-[Kompose](https://github.com/kubernetes-incubator/kompose) is a tool to help users familiar with Docker Compose
+[Kompose](https://github.com/kubernetes-incubator/kompose) is a tool to help users be familiar with Docker Compose
```

Comment:
The original wording isn't wrong. It helps users that are already familiar with Docker Compose, not helps users to become familiar with Docker Compose. 

Maybe we can rephrase this to something like "...to help Docker Compose users move to Kubernetes"?